bundles/remote_services/remote_service_admin_dfi/src/export_registration_dfi.c

```diff
@@ -86,6 +86,7 @@ celix_status_t exportRegistration_create(celix_log_helper_t *helper, service_ref
         reg->logFile = logFile;
         reg->servId = strndup(servId, 1024);
         reg->trackerId = -1L;
+        reg->active = true;
```

Comment:
Not sure if it is relevant at this point, but the function setActive uses a lock, is that needed here as well?

Reply:
in the create the lock is not needed yet; the structure is then only known to a single thread (the creating the structure).